recipes/cffi/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cffi" %}
+{% set version = "1.7.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "6ed5dd6afd8361f34819c68aaebf9e8fc12b5a5893f91f50c9e50c8886bb60df" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+  patches:
+    - setup-linux.patch  # [linux]
```

Comment:
Not sure about the reason, but the recipes http://repo.continuum.io/pkgs/free follow the same pattern (only for linux), I will need to ask